pvnet_app/app.py

```diff
@@ -305,18 +292,14 @@ def app(
         national_capacity = get_latest_gsp_capacities(session, [0])[0]
 
     # Download satellite data
-    if use_satellite:
-        logger.info("Downloading satellite data")
-        download_all_sat_data()
-
-        # Preprocess the satellite data and record the delay of the most recent non-nan timestep
-        all_satellite_datetimes, data_freq_minutes = preprocess_sat_data(
-            t0, 
-            use_legacy=use_day_ahead_model
-        )
+    logger.info("Downloading satellite data")
```

Comment:
I added in this use satellite, as it didn't work when there was no satellite data. 

I wonder if there is an easy way to add this functionality back in

Reply:
Actually maybe you keep this in